syne_tune/blackbox_repository/conversion_scripts/scripts/yahpo_import.py

```diff
@@ -163,15 +166,22 @@ def _adjust_fidelity_space(self, fidelities: Optional[List[int]]):
             self._fidelity_values = np.array(fidelities)
             self.fidelity_space[self._fidelity_name] = cs.ordinal(fidelities.copy())
 
+    def active_hyperparameters(self, configuration: dict) -> List[str]:
+        return self.benchmark.config_space.get_active_hyperparameters(
+            ConfigSpace.Configuration(
```

Comment:
I am not sure whether removing ConfigSpace would be easy as 1) it would require to rewrite entirely the wrapper 2) you would still need in your dependency to import yahpo. Also note that this PR does not add a dependency on ConfigSpace it was imported/used before.